python-package/lightgbm/basic.py

```diff
@@ -3283,6 +3293,10 @@ def model_from_string(self, model_str, verbose=True):
         _safe_call(_LIB.LGBM_BoosterGetNumClasses(
             self.handle,
             ctypes.byref(out_num_class)))
+        if verbose != 'warn':
+            _log_warning("'verbose' argument is deprecated and will be removed in 4.0.0 release.")
+        else:
+            verbose = True
         if verbose:
             _log_info(f'Finished loading model, total used {int(out_num_iterations.value)} iterations')
```

Comment:
For this one, I don't think one log line deserves a standalone argument. Users are able to get number of iterations after constructing a Booster if needed.

Reply:
agree! (sorry I missed commenting on this in my previous review)